### DISCUSSION TITLE
Avalon Template: Drawer menu open by default

### DISCUSSION THREAD

**Discussion by lonnieCross4568** [primefaces/community#597] [PrimeNG Templates] [answered]
I'm trying to have the drawer menu open by default. I've tried to set the layoutService.state.sidebarActive to true and toggle the anchor, but when I do that the drawer no longer functions.

**Comment by cetincakiroglu** [comment, marked as answer]
You don't need to change it by state. Simply you can add the classes attached by selecting drawer mode. Navigate to the `app.layout.component.html` and insert the classes `layout-container layout-topbar-purple layout-menu-light layout-menu-profile-start layout-drawer layout-sidebar-active layout-sidebar-anchored` to the div with the class `layout-container`. Remove the `[ngClass]="containerClass"`
